vip_cram.nf

```diff
@@ -69,6 +106,11 @@ def parseSampleSheet(csvFile) {
       type: "string",
       default: { 'illumina' },
       enum: ['illumina', 'nanopore']
+    ],
+    analysis_type: [
```

Comment:
To me 'analysis_type' implies the type of analysis you want to perform instead of describing the type of data. It doesn't make sense to do a WGS analysis on WES data. 'sequencing_method' would be a better term in my opinion.